From ce615ee6324b28701e389d5ca454902320bca2cd Mon Sep 17 00:00:00 2001
From: Arun Bharadwaj <arun@gumstix.com>
Date: Wed, 15 Apr 2015 16:24:04 -0700
Subject: [PATCH 19/57] gpio-backlight: Discover driver during boot time

The gpio-backlight driver seems to be missing the
MODULE_DEVICE_TABLE line which is preventing it from
being modprobed during boot time even if the
gpio-backlight device exists.

This seems to be a bug and this patch attempts to
fix that.

Signed-off-by: Arun Bharadwaj <arun@gumstix.com>
Signed-off-by: Ash Charles <ashcharles@gmail.com>
---
 drivers/video/backlight/gpio_backlight.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/video/backlight/gpio_backlight.c b/drivers/video/backlight/gpio_backlight.c
index 439feb2..5fbbc2e 100644
--- a/drivers/video/backlight/gpio_backlight.c
+++ b/drivers/video/backlight/gpio_backlight.c
@@ -146,6 +146,8 @@ static struct of_device_id gpio_backlight_of_match[] = {
 	{ .compatible = "gpio-backlight" },
 	{ /* sentinel */ }
 };
+
+MODULE_DEVICE_TABLE(of, gpio_backlight_of_match);
 #endif
 
 static struct platform_driver gpio_backlight_driver = {
-- 
2.7.4

